### ISSUE TITLE
Unable to update the document with numeric PartitionKey

### ISSUE THREAD

**Issue by mehul115** [PlagueHO/CosmosDB#316]
# Unable to update the document with numeric PartitionKey

When partition keys are numeric, the Set-CosmosDbDocument function fails as it sends the partition key header as `["123"]` instead of `[123]`. Below is the snippet of the code which needs to be updated to extend the support for numeric partition keys.

```powershell
    if ($PSBoundParameters.ContainsKey('PartitionKey'))
    {
        $headers += @{
            'x-ms-documentdb-partitionkey' = '["' + ($PartitionKey -join '","') + '"]'
        }
        $null = $PSBoundParameters.Remove('PartitionKey')
    }
```
Below are the environment details:
- [ ] PSVersion: 6.2.3
- [ ] Console Host
- [ ] Windows 10
- [ ] CosmosDB Module Version: 3.5.0.426

**Comment by PlagueHO** [comment]
Hi @mehul115 - there is actually a PR for this in process: https://github.com/PlagueHO/CosmosDB/pull/311

I've just got to get it reopened by @Splaxi as I accidentally merged it before reviewing so had to revert.

**Comment by PlagueHO** [comment]
HI @mehul115 - I ended up pushing this one out myself. I should have this new version released to PSGallery in the next 24 hours.

**Comment by PlagueHO** [comment]
Hi @mehul115 - this has been published the PSGallery in 3.5.1:
https://www.powershellgallery.com/packages/CosmosDB/3.5.1.458
